data/companies.yml

```diff
@@ -284,6 +284,12 @@ integrations:
     description: "Space Cloud is a Kubernetes based serverless platform that provides instant, realtime APIs on any database, with event triggers and unified APIs for your custom business logic."
     details:
       - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices."
+  - name: "CSFMesh"
+    logo: "/logos/asiainfo.png"
+    url: "https://www.asiainfo.com/zh_cn/product_aisware_aifPaaS.html"
+    description: "CSFMesh is a service mesh product developed by AsiaInfo Technologies. It features a graphical configuration interface, which enhances operational efficiency."
+    details:
```

Comment:
Your text in desc and details shall clearly show the connections to Istio.

Reply:
i already add more details for show connection to istio